gno.land/cmd/gnoland/secrets_common.go

```diff
@@ -145,7 +145,7 @@ func validateValidatorStateSignature(
 }
 
 // validateNodeKey validates the node's p2p key
-func validateNodeKey(key *p2p.NodeKey) error {
+func validateNodeKey(key *types.NodeKey) error {
```

Comment:
even if p2p/types is the module name, the usage should not be "types" unless it is within the same parent module like within p2p. cmd/gnoland/secrets_common.go is outside of p2p, so it should be p2p, unless there's a reason why it can't be, then it could be p2ptypes, but why can't it be p2p?